modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml

```diff
@@ -53,7 +53,7 @@
 	</methods>
 	<members>
 		<member name="delta_interval" type="float" setter="set_delta_interval" getter="get_delta_interval" default="0.0">
-			Time interval between delta synchronizations. When set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
+			Time interval between delta synchronizations. Useful when the replication is set to [code]On Change[/code]. A value of [code]0.0[/code] (the default) means that delta synchronizations happen every network process frame.
```

Comment:
```suggestion
			Time interval between delta synchronizations. Used when the replication is set to [code]On Change[/code]. A value of [code]0.0[/code] (the default) means that delta synchronizations happen every network process frame.
```

We can't really mind read to know what is useful so I was proposing Useful -> Used.